src/pycountry/__init__.py

```diff
@@ -284,21 +302,21 @@ def add_result(
 
 
 # Initialize instances with type hints
-countries: ExistingCountries = ExistingCountries(
```

Comment:
The typing isn't needed since it's defined from the right hand side of the assignment